packages/flutter_tools/lib/src/ios/application_package.dart

```diff
@@ -145,6 +145,10 @@ class BuildableIOSApp extends IOSApp {
   String get archiveBundleOutputPath =>
       globals.fs.path.setExtension(archiveBundlePath, '.xcarchive');
 
+  String get builtInfoPlistPathAfterArchive => globals.fs.path.join(archiveBundleOutputPath,
+      'Products', 'Applications', globals.fs.path.setExtension(_hostAppBundleName == null ? 'Runner' : globals.fs.path.withoutExtension(_hostAppBundleName!), '.app'),
```

Comment:
I think `_hostAppBundleName` has the `.app` so is there any reason to strip off the extension and re-add it?

Reply:
Ah i thought it's `.bundle` files. will update